README.md

```diff
@@ -293,165 +317,78 @@ export default MessageList = () => {
-  - `autoRenew` *(optional)*:
-  By default, the library will attempt to renew expired tokens. When an expired token is requested by the library, a renewal request is executed to update the token. If you wish to  to disable auto renewal of tokens, set autoRenew to false.
-  - `secure`: If `true` then only "secure" https cookies will be stored. This option will prevent cookies from being stored on an HTTP connection. This option is only relevant if `storage` is set to `cookie`, or if the client browser does not support `localStorage` or `sessionStorage`, in which case `cookie` storage will be used.
-  - `storage` *(optional)*:
-    Specify the type of storage for tokens.
-    The types are:
-    - [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
-    - [`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
-    - [`cookie`](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie)
-
-#### Example
-```jsx
-function customAuthHandler(authService) {
-  // Redirect to the /login page that has a CustomLoginComponent
-  // This example is specific to React-Router
-  this.props.history.push('/login');
-}
-
-class App extends Component {
-  render() {
-    return (
-        <Security issuer='https://{yourOktaDomain}.com/oauth2/default'
-                  clientId='{clientId}'
-                  redirectUri={window.location.origin + '/implicit/callback'}
-                  onAuthRequired={customAuthHandler} >
-          <Router path='/login' component={CustomLoginComponent}>
-          {/* some routes here */}
-        </Security>
-    );
-  }
-}
-
-export default withRouter(App);
-```
+`<Security>` is the top-most component of okta-react. It accepts [oktaAuth][Okta Auth SDK] instance and addtional configuration options as props.
 
-#### Alternate configuration using `AuthService` instance
+#### oktaAuth
 
-When the `authService` option is passed, all other configuration options passed to `Security` will be ignored.
+*(required)* The pre-initialized [oktaAuth][Okta Auth SDK] instance. See [Configuration Reference](https://github.com/okta/okta-auth-js#configuration-reference) for details of how to initialize the instance.
 
-- **authService** *(optional)* - Provide an [Auth service][] instance instead of the options above. This is the most direct way to use methods on the [Auth service][] instance *outside* of your components and is helpful when integrating `okta-react` with external libraries that need access to the tokens.
+#### onAuthRequired
 
-#### Example with AuthService object
+*(optional)* Callback function. Called when authentication is required. If this is not supplied, `okta-react` redirects to Okta. This callback will receive [oktaAuth][Okta Auth SDK] instance as the first function parameter. This is triggered when a `SecureRoute` is accessed without authentication.
```

Comment:
Can we give better advice on what this should be used for?  We discuss when it is called, but not why.  Our example below sends them to a location - is this instead of the normal redirection?  We don't say what happens in such a case here.